src/FixedFrequencyAgent.cpp

```diff
@@ -80,24 +74,15 @@ namespace geopm
 
     void FixedFrequencyAgent::init_platform_io(void)
     {
-        // Populate signals for each domain with batch idx info, default values, etc
-        for (auto &sv : m_signal_available) {
-            sv.second.m_batch_idx = m_platform_io.push_signal(sv.first, GEOPM_DOMAIN_BOARD, 0);
-        }
-
-        // Populate controls for each domain
-        for (auto &sv : m_control_available) {
-            sv.second.m_batch_idx = m_platform_io.push_control(sv.first, GEOPM_DOMAIN_BOARD, 0);
-        }
 
     }
 
     // Validate incoming policy and configure default policy requests.
     void FixedFrequencyAgent::validate_policy(std::vector<double> &in_policy) const
     {
         assert(in_policy.size() == M_NUM_POLICY);
-        double accel_min_freq = m_platform_io.read_signal("NVML::FREQUENCY_MIN", GEOPM_DOMAIN_BOARD, 0);
-        double accel_max_freq = m_platform_io.read_signal("NVML::FREQUENCY_MAX", GEOPM_DOMAIN_BOARD, 0);
+        double accel_min_freq = m_platform_io.read_signal("NVML::GPU_FREQUENCY_MIN_AVAIL", GEOPM_DOMAIN_BOARD, 0);
+        double accel_max_freq = m_platform_io.read_signal("NVML::GPU_FREQUENCY_MAX_AVAIL", GEOPM_DOMAIN_BOARD, 0);
```

Comment:
If rebasing on top of PR #1945 these can be the aliases: 
GPU_FREQUENCY_MIN_AVAIL
GPU_FREQUENCY_MAX_AVAIL
without the NVML:: prefix.

Reply:
Pulled in latest dev and uses aliases